lib/active_merchant/billing/gateways/cecabank/cecabank_json.rb

```diff
@@ -196,7 +196,7 @@ def add_stored_credentials(post, creditcard, options)
 
       def add_three_d_secure(post, options)
         params = post[:parametros] ||= {}
-        return params[:ThreeDsResponse] = '{}' unless three_d_secure = options[:three_d_secure]
+        return unless three_d_secure = options[:three_d_secure]
```

Comment:
💬 Comment:

I think that would be great to have a unit tests that ensures that `ThreeDsResponse` is not present on the request if there is no content on the `options[:three_d_secure]`

Reply:
Thanks! New test added